site/contents/upcoming-events.md

```diff
@@ -0,0 +1 @@
+## Upcoming Events
```

Comment:
I plan to eventually have a calendar here, but baby steps (doesn't currently show up in TOC)